banking-api/src/main/java/com/banking/api/BankAccountOpenFeignClient.java

```diff
@@ -1,30 +1,31 @@
 package com.banking.api;
 
 import com.banking.dto.BankAccountDto;
-import java.util.List;
 import java.util.UUID;
-import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
+import reactivefeign.spring.config.ReactiveFeignClient;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
-@FeignClient(name = "banking-microservice")
+@ReactiveFeignClient(name = "banking-microservice")
```

Comment:
great! But how it works with multi-pods?

Reply:
As I understood, eureka returns all pods of banking-microservice and the OpenFeign will balance load between existing pods.